### PR TITLE
fix: fix ci

### PR DESCRIPTION
**Pull request by whxaxes**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

fix ci for https://github.com/eggjs/egg-passport/pull/8

### CODE REVIEW

File: test/app-ts.test.js [hunk 1]
```diff
@@ -31,15 +31,6 @@ describe('test/app-ts.test.js', () => {
 
     after(async () => {
       await app.close();
-      assert.deepStrictEqual(app._app.stages, [
```
Comment (whxaxes) [inline test/app-ts.test.js:34]: 这个 assert 没意义，而且也没有 stages 这个值，所以删了

File: test/fixtures/app-ts/app.ts [hunk 2]
```diff
@@ -12,8 +13,9 @@ interface UserIdentity {
 }
 
 export default (app: Application) => {
-  app.passport.verify(async (ctx, user: UserIdentity) => {
+  app.passport.verify(async (_ctx, user: UserIdentity) => {
     if (user.provider === 'localapikey') {
+      console.info(user.apikey);
```
Comment (whxaxes) [inline test/fixtures/app-ts/app.ts]:
```suggestion

```

**Review by thonatos — APPROVED** [review]

**Comment by whxaxes** [comment]
@thonatos  发个版本？

**Comment by atian25** [comment]
@whxaxes 给你加权限了

**Comment by whxaxes** [comment]
+ egg-passport@2.1.0

**Pull request merged**